TombEngine/Objects/Generic/Switches/airlock_switch.cpp

```diff
@@ -0,0 +1,82 @@
+#include "framework.h"
+#include "Game/control/control.h"
+#include "Specific/Input/Input.h"
+#include "Specific/level.h"
+#include "Game/Lara/lara.h"
+#include "Game/Lara/lara_helpers.h"
+#include "Objects/Generic/Switches/generic_switch.h"
+#include "Game/animation.h"
+#include "Game/collision/collide_item.h"
+#include "Game/items.h"
+
+using namespace TEN::Input;
+
+namespace TEN::Entities::Switches
+{
+	ObjectCollisionBounds LockSwitchBounds =
+	{
+		GameBoundingBox::Zero,
+		std::pair(
+			EulerAngles(ANGLE(-10.0f), ANGLE(-30.0f), ANGLE(-10.0f)),
+			EulerAngles(ANGLE(10.0f), ANGLE(30.0f), ANGLE(10.0f))
+		)
+	};
+
+	auto AirLockSwitchPos = Vector3i::Zero;
```

Comment:
File elects "airlock" spelling variation, so let's do `Airlock` here as well and for functions.

Reply:
Changed AirLock to Airlock.

I let the capital start because although these, (AirlockSwitchBounds and AirlockSwitchPos) are used only here, they are still global.